invertible_layers.py

```diff
@@ -234,8 +234,8 @@ def reverse_(self, x, objective):
 class AdditiveCoupling(Layer):
     def __init__(self, num_features):
         super(AdditiveCoupling, self).__init__()
-        assert num_features % 2 == 0
-        self.NN = NN(num_features // 2)
+        # assert num_features % 2 == 0
+        self.NN = NN(math.ceil(num_features / 2), channels_out=math.floor(num_features / 2) * 2)
```

Comment:
hey, looking at the old code it seems that `self.NN` took `num_features // 2` channels in and returned `num_features // 2` channels out. However it the new code the number of output channels seems to be doubled?

Reply:
Ah, sure, this problem is for the additive coupling layer. I will correct it. The affine looks fine yes?

Reply:
I have changed the code for additive. Does it look fine now?

Reply:
looks good. I'm assuming you checked that it works now for both odd and even n_channels ?

Reply:
I am using it in my own work right now and it works. but I have changed a few things in my code too. So please give me a little time to also test it on your own code without my changes and I will leave another comment as soon as I did it. So, I think do not accept the pull request right now so I can be 100% sure.

Reply:
sounds good, there's no rush. thanks for the PR!